src/main/java/com/mraof/minestuck/world/gen/feature/tree/EndTreeFeature.java

```diff
@@ -32,7 +33,7 @@ public boolean place(FeaturePlaceContext<NoneFeatureConfiguration> context)
 		WorldGenLevel level = context.level();
 		BlockPos position = context.origin();
 		BlockPos soilPos = position.below();
-		if(level.getBlockState(soilPos).canSustainPlant(level, soilPos, Direction.UP, MSBlocks.END_SAPLING))
+		if(level.getBlockState(soilPos).canSustainPlant(level, soilPos, Direction.UP, (IPlantable) MSBlocks.END_SAPLING.get()))
```

Comment:
Remove the need for this new cast by changing the type of `END_SAPLING` to `RegistryObject<BushBlock>` to match the block type that `END_SAPLING` had before.

Reply:
fixed